orchagent/mirrororch.cpp

```diff
@@ -1342,7 +1342,10 @@ void MirrorOrch::updateLagMember(const LagMemberUpdate& update)
             // If LAG is empty, deactivate session
             if (update.lag.m_members.empty())
             {
-                deactivateSession(name, session);
```

Comment:
Is it requested by end user? If yes, the error message is expected.

Reply:
It seems that the session got deactivated twice, and we should avoid deactivating the same session once it is already deactivated. I think checking the session status should be a good way to do the job. The function `deactivateSession()` even has an assertion that session status is true at the beginning of the function.